README.rst

```diff
@@ -217,6 +224,7 @@ A sample config file:
     ci:
       github:
         path: '{wf_name}/{number}/'
+        artifacts_path: '{wf_name}/{number}-artifacts/'
```

Comment:
nice... just a note: but for "releases" we would need an additional/separate `artifacts_path` specification (will not have wf_name etc)